include/evmc/evmc.hpp

```diff
@@ -150,6 +150,110 @@ constexpr bytes32::operator bool() const noexcept
     return !is_zero(*this);
 }
 
+namespace literals
+{
+namespace internal
```

Comment:
Should internal stuff better go into separate header?

Reply:
So far I'm ok with having single header for C++ (now it's even better when helpers.hpp is gone).